### ISSUE TITLE
Asserting unlocked bootloader variable

### ISSUE THREAD

**Issue by DragonLych** [ubports/ubports-installer#2816]
**UBports Installer `0.9.6-beta` (exe)**
Environment: `Microsoft Windows 10 Pro 10.0.19044 Windows 10.0.19044 x64 19044 0.0 NodeJS v16.14.2`
Device: `FP4`
Target OS: undefined
Settings: `{}`
OPEN-CUTS run: *N/A* 
Pastebin: https://snip.hxrsh.in/ubi-1661631780484

Installing Fairphone 4 image, successfully get to the asserting unlocked bootloader variable but cannot go any further. Device is definitely fully unlocked. Same issue occurs when trying to install from Linux Mint Host

**Comment by Termina94** [comment]
Having the same issue here trying on Windows and Fedora.

- Bootloader fully unlocked
- Fresh install of fairphone OS
- developer mode and USB debugging enabled
- fastboot bootloader says "DEVICE STATE - unlocked"

**Comment by Termina94** [comment]
Temporary solution that fixed my issue was bypassing the check that hangs.

Download this file: https://github.com/ubports/installer-configs/blob/master/v2/devices/FP4.yml

Or copy the contents to a file eg: config.yml

Remove or comment out the lines:

```
      - actions:
          - fastboot:assert_var:
              variable: "unlocked"
              value: "yes"
```

Then run

```
ubports-installer -f ./config.yml
```

Follow installer like normal and it will use your new local config

This skipped the check and installed successfully on my FP4

**Comment by amartinz** [comment]
We have just released 0.9.7-beta.
Could you please retry with this one?

**Comment by DragonLych** [comment]
That's worked! Much appreciated
